skpro/distributions/gamma.py

```diff
@@ -0,0 +1,78 @@
+# copyright: skpro developers, BSD-3-Clause License (see LICENSE file)
+"""Exponential probability distribution."""
+
+__author__ = ["ShreeshaM07"]
+
+import pandas as pd
+from scipy.stats import gamma, rv_continuous
+
+from skpro.distributions.adapters.scipy import _ScipyAdapter
+
+
+class Gamma(_ScipyAdapter):
+    r"""Gamma Distribution.
+
+    The Gamma Distribution is parameterized by shape :math:`\alpha` and
+    rate :math:`\beta`, such that the pdf is
+
+    ..math:: f(x) = \frac{x^(\alpha-1)*\exp\left(-\beta*x)*\beta^\alpha}{\tau(\alpha)}
+
+    where `\tau(\alpha)` is the Gamma function.
+    For all positive integers,`\tau(\alpha) = (alpha-1)!`.
```

Comment:
```suggestion
    where :math:`\tau(\alpha)` is the Gamma function.
    For all positive integers, :math:`\tau(\alpha) = (\alpha-1)!`.
```

Reply:
Could you please be so kind to write the expression for the `f(x)` as well in this LaTex format I am not able to do it with trial and error, or maybe lead me to some resource that explains the usage.

Reply:
but `f` looks correct already - does it not compile to sth readable?

You can always test it by putting it in markdown, like this:

$`f(x) = \frac{x^{\alpha-1}\exp\left(-\beta x\right)\beta^{\alpha}}{\tau(\alpha)}`$

Reply:
for some reason the expression is working when i use $$ in markdown here but it doesnt convert in readthedocs and instead if i use ..math:: neither here nor there it is readable.

Reply:
I'm looking at the docs, it is perfectly readable for me:

![image](https://github.com/sktime/skpro/assets/7985502/4691b467-979e-4296-b5a1-f547a1d8f39b)